.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,9 @@
  your pull request. The description should explain what will change,
  and why.
 
+ PLEASE title the FIRST commit appropriately, so that when the pull
+ request is merged and squashed, the commit message makes sense.
+
```

Comment:
@onlydole this repo now uses merge-method-merge as the default (previously it was merge-method-squash).

Is this advice still right? I agree about the message, but it might imply that the repo still uses merge-method-squash.

Reply:
```suggestion
 PLEASE title the FIRST commit appropriately, so that if you squash all
 your commits into one, the combined commit message makes sense.
```
?